src/block-components/alignment/editor-has-polyfill.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { useBlockContext } from '~stackable/hooks'
+
+import { select } from '@wordpress/data'
+import { useBlockEditContext } from '@wordpress/block-editor'
+
+export const EditorHasPolyfill = () => {
```

Comment:
The case of this one is usually used for Components, use `editorHasPolyfill` instead